src/sentry/api/event_search.py

```diff
@@ -176,6 +176,8 @@ class SearchVisitor(NodeVisitor):
     # A list of mappers that map source keys to a target name. Format is
     # <target_name>: [<list of source names>],
     key_mappings = {}
+    numeric_keys = set()
```

Comment:
should the `issue.id` key be numeric?

Reply:
What does that actually map to in snuba, just the `group_id` column?

Reply:
Actually, we probably don't want it to be a numeric column, at least for now, since those allow `>`, `<` etc. And I assume these worked ok before we had the numeric column, I guess the input is cast at some point.